scene/2d/area_2d.cpp

```diff
@@ -401,10 +398,7 @@ void Area2D::set_monitoring(bool p_enable) {
 
 	if (p_enable == monitoring)
 		return;
-	if (locked) {
-		ERR_EXPLAIN("Function blocked during in/out signal. Use set_deferred(\"monitoring\",true/false)");
-	}
-	ERR_FAIL_COND(locked);
+	ERR_FAIL_COND_MSG(locked, "Function blocked during in/out signal. Use set_deferred(\"monitoring\",true/false)");
```

Comment:
Nitpick, but while we're changing this, can you add a space after `"monitoring",`? Same below.